sdks/python/apache_beam/transforms/enrichment_handlers/vertex_ai_feature_store_it_test.py

```diff
@@ -75,7 +76,7 @@ def setUp(self) -> None:
     self._start_container()
 
   def _start_container(self):
-    for i in range(3):
```

Comment:
can we use self.retries here? similar to other files - thanks

Reply:
Done